.buildkite/hooks/pre-command

```diff
@@ -4,19 +4,15 @@ source .buildkite/scripts/tooling.sh
 
 set -euo pipefail
 
-GITHUB_TOKEN_VAULT_PATH=kv/ci-shared/platform-ingest/github_token
-
 GO_VERSION=$(cat .go-version)
 export GO_VERSION
 
 # Secrets must be redacted
 # https://buildkite.com/docs/pipelines/managing-log-output#redacted-environment-variables
 
 if [[ "$BUILDKITE_PIPELINE_SLUG" == "package-spec-test-with-integrations" && "$BUILDKITE_STEP_KEY" == "pr-integrations" ]]; then
-    GITHUB_USERNAME_SECRET=$(retry 5 vault kv get -field username ${GITHUB_TOKEN_VAULT_PATH})
-    export GITHUB_USERNAME_SECRET
-    GITHUB_EMAIL_SECRET=$(retry 5 vault kv get -field email ${GITHUB_TOKEN_VAULT_PATH})
-    export GITHUB_EMAIL_SECRET
-    GITHUB_TOKEN=$(retry 5 vault kv get -field token ${GITHUB_TOKEN_VAULT_PATH})
-    export GITHUB_TOKEN
+    GITHUB_USERNAME_SECRET="elasticmachine"
+    export GITHUB_USERNAME_SECRET=$GITHUB_USERNAME_SECRET
+    export GITHUB_EMAIL_SECRET="elasticmachine@elastic.co"
+    export GITHUB_TOKEN=$VAULT_GITHUB_TOKEN
```

Comment:
If ephemeral tokens are used, maybe it's not needed those environment variables.

Are those ephemeral tokens already available for `git` commands ?

If they are, we could try to remove these functions (and respective calls) and test it:
https://github.com/elastic/package-spec/blob/7aa453907fb0406e32dca37e296e0a2199416487/.buildkite/scripts/test-with-integrations.sh#L56-L67


The same would happen for the elastic-package: https://github.com/elastic/elastic-package/pull/1942

Reply:
I tried to remove this git config lines and test it, but it is not work and ask to set this parameters.
So I will leave it like it is for now.